sql/parser/eval.go

```diff
@@ -591,13 +726,6 @@ func init() {
 // the client.
 func EvalExpr(expr Expr) (Datum, error) {
 	switch t := expr.(type) {
-	case Row:
```

Comment:
Add a `case` for `Row` that indicates that `// NormalizeExpr transforms this into Row`.